build/Dockerfile.builder

```diff
@@ -26,7 +26,7 @@ RUN ln -s /usr/bin/microdnf /usr/bin/dnf \
 
 USER root
 
-ENV OPERATOR_SDK_VERSION 0.15.1
+ENV OPERATOR_SDK_VERSION 0.18.2
```

Comment:
Does this require any update with the `yq` package as well?

Reply:
No the current yq package we have been using (3.2.1) works without issue.